CHANGELOG.md

```diff
@@ -0,0 +1,29 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.3.0] - 2021-11-09
+### Added
```

Comment:
Nothing was fixed during all the releases? :)

Reply:
Now something is fixed. We saved the children! :)